src/table.mjs

```diff
@@ -254,14 +254,28 @@ export function makeQueryTemplate(operations, source) {
     appendSql(i ? `, ` : `\nORDER BY `, args);
     appendOrderBy(sort[i], args);
   }
-  if (slice.to !== null || slice.from !== null) {
-    appendSql(
-      `\nLIMIT ${slice.to !== null ? slice.to - (slice.from || 0) : 1e9}`,
-      args
-    );
-  }
-  if (slice.from !== null) {
-    appendSql(` OFFSET ${slice.from}`, args);
+  if(source.dialect === 'mssql'){
+    if (slice.to !== null || slice.from !== null) {
+      if(!sort.length){
+        appendSql(`\nORDER BY `, args);
+        appendOrderBy({column: select.columns[0], direction: 'ASC'}, args);
```

Comment:
I think we’ll want the escaped column name here, to match the SELECT.

```suggestion
        appendOrderBy({column: columns[0], direction: 'ASC'}, args);
```

Also, this will crash if select.columns is null, as it is with `SELECT *`. In practice this should never happen for a MS SQL data source because we would always materialize the selected columns in a table cell, but we should either throw an error here with a human-readable message or perhaps we can make it work with something like ORDER BY NULL.